src/HPWH.cc

```diff
@@ -444,7 +444,7 @@ HPWH& HPWH::operator=(const HPWH& hpwh)
         return *this;
     }
 
-    Sender::operator = (hpwh);
+    Sender::operator=(hpwh);
```

Comment:
TODO: Check if this is necessary.

Reply:
Would seem to be needed if assignment of HPWH objects is performed. (However, unlike HeatSources, HPWHs are instantiated with a default courier, so no fault will occur.)

Reply:
Makes sense. Thanks!